tests/test_help.py

```diff
@@ -27,6 +27,7 @@ def test_help(self, mocker: Any) -> None:
         mocker.patch('git_machete.utils.run_cmd', mock_run_cmd)  # to hide git outputs in tests
 
         expected_exit_code = None
+        help_topics: List[str] = ['config']
```

Comment:
I'd say that `format` and `hooks` are also _help topics_ (and not commands)... why just `config` is included here? 🤔

Reply:
Well, it turns out that both `format` and `hooks` are actually commands that do nothing :D

`    subparsers.add_parser(
        'format', add_help=False, usage=argparse.SUPPRESS, parents=[common_args_parser])`

`    subparsers.add_parser(
        'hooks',
        usage=argparse.SUPPRESS,
        add_help=False,
        parents=[common_args_parser])`
@PawelLipski i should remove them, right?